invenio/b2share/modules/b2deposit/b2share_model/metadata/aalto_metadata_config.py

```diff
@@ -0,0 +1,73 @@
+from linguistics_lang_codes import lang_codes
+from invenio.ext.sqlalchemy import db
+
+domain = 'Aalto'
+# display_name = 'Research Data Alliance'
+display_name = 'Aalto'
+table_name = 'Aalto'
+image = 'domain-aalto.jpg'
+kind = 'project'
+domaindesc = 'Aalto University'
+
+# the domain administators can edit even published records
+admin_can_edit_published_record = True
+
+# only the domain administrators can deposit
+depositing_groups = ['aalto_domain_administrators']
```

Comment:
Ok, that sounds good. So it would be like: depositing_groups = ['aalto_domain_administrators','aalto_domain_members'] ?